src/adv.py

```diff
@@ -38,14 +41,56 @@
 #
 
 # Make a new player object that is currently in the 'outside' room.
+print('Welcome to this crazy game!  Enter a name for you player...')
+name = input('>>')
+player = Player(name, room['outside'], None)
 
-# Write a loop that:
-#
-# * Prints the current room name
-# * Prints the current description (the textwrap module might be useful here).
-# * Waits for user input and decides what to do.
-#
-# If the user enters a cardinal direction, attempt to move to the room there.
-# Print an error message if the movement isn't allowed.
-#
-# If the user enters "q", quit the game.
+
+def current_room(player):
+    print(f'{player.name} is in {player.room}')
+    if player.room.items == 'no items':
+        print('this room has no items')
+    else:
+        print(f'this room has a{player.room.items}')
+        print(
+            'What would you like to do? [GET] pick up [N] North [S] South [E] East [W] West [Q] Quit ')
+        playerInput = input(">>").upper()
+
+
+playerInput = ''
+while playerInput != 'Q':
+    print(current_room(player))
+    print('--------------------')
+    print(player)
+    print(
+        'Choose your next move: [N] North [S] South [E] East [W] West [Q] Quit')
+    playerInput = input(">>").upper()
+
```

Comment:
The program crashes when attempting to move an invalid direction. Do some more testing and see if you can figure out why